### ISSUE TITLE
[FEAT] Dark mode missing

### ISSUE THREAD

**Issue by aurarius1** [linuxserver/docker-orcaslicer#9]
### Is this a new feature request?

- [X] I have searched the existing issues

### Wanted change

OrcaSlicer on desktop has a dark mode, is this feature disabled for the container?

### Reason for change

Because OrcaSlicer on desktop has it.

**Comment by XxSimpleAceXx** [comment]
+1 darkmode would be nice to see implemented

**Comment by arraylabs** [comment]
This would be great!

**Comment by machetie** [comment]
please add this feature, the only thing that stops me using it in docker.

**Comment by ss1gohan13** [comment]
+1 for dark mode

**Comment by j0nnymoe** [comment]
https://github.com/SoftFever/OrcaSlicer/issues/883

It doesn't seem to be an option within the appimage version of OrcaSlicer, seems it has to be modified via the DE settings.

**Comment by ss1gohan13** [comment]
> [SoftFever/OrcaSlicer#883](https://github.com/SoftFever/OrcaSlicer/issues/883)
> 
> It doesn't seem to be an option within the appimage version of OrcaSlicer, seems it has to be modified via the DE settings.

Could you elaborate into this a little bit for me?

**Comment by AndreaPro** [comment]
> please add this feature, the only thing that stops me using it in docker.

Same!

**Comment by j0nnymoe** [comment]
> > [SoftFever/OrcaSlicer#883](https://github.com/SoftFever/OrcaSlicer/issues/883)
> > It doesn't seem to be an option within the appimage version of OrcaSlicer, seems it has to be modified via the DE settings.
> 
> Could you elaborate into this a little bit for me?

Not really, it's been mentioned within that thread that it isn't part of their app image release.

**Comment by nsyring** [comment]
This is a great Image. I also would appreciate darkmode. I think this has to be done in the openbox-settings paired with some theme-package installs.

**Comment by j0nnymoe** [comment]
The current solution is janky at best. This should just be added upstream.

**Comment by wernight** [comment]
To use dark themed, set `-e GTK_THEME=Adwaita:dark`  or the equivalent environment in `docker-compose.yml`.

**Comment by AndreaPro** [comment]
> To use dark themed, set `-e GTK_THEME=Adwaita:dark` or the equivalent environment in `docker-compose.yml`.

It works!
